curvefs/src/client/s3/disk_cache_read.cpp

```diff
@@ -102,7 +102,8 @@ int DiskCacheRead::LinkWriteToRead(const std::string fileName,
     return 0;
 }
 
-int DiskCacheRead::LoadAllCacheReadFile(std::set<std::string> *cachedObj) {
+int DiskCacheRead::LoadAllCacheReadFile(
```

Comment:
Never find diskcache read hit the cache!

Reply:
IsCache is been called before Read.